Update screenshots/captions for new mobile UI (#12208)
newrelic/docs-website: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/crash-analysis-group-filter-your-crashes.mdx

    title="Examine crash report details: Export to Xcode or resymbolicate your source code."
  >
    From the **Crash types** table, you can drill down into a specific [crash type](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/investigate-mobile-app-crash-report). From here you can:

>     * Examine the [interaction trail](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/interactions-page) from session start through the crash event.

Author: Has this feature been removed or just moved in the Mobile UI?

Reviewer: We removed this specific component. In the event trail (below the stack trace) a user can click on the Interaction icon and limit the event trail to interactions, which gives the same information as the interaction trail used to give.

>     * Resymbolicate the crash report for easier debugging.

Author: ```suggestion
```

>     * File a ticket and resolve the crash.

Author: ```suggestion
    * Click the **View crash libraries** button to see what Android map files or iOS .dSYM files we have for your application.
    * If you don't see the files you're expecting, upload a new Android map file or iOS .dSYM files by clicking on the **Upload** link. Note that uploading a new file automatically de-obfuscates or symbolicate your crash occurrence.
    * File a ticket and resolve the crash.
```

      <img
        title="New Relic Mobile Crash Analytics: Stack trace"
        alt="New Relic Mobile Crash Analytics: Stack trace"
        src={mobileStackTrace}
      />

      <figcaption>

>         **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Crashes > Crash type > (select a crash) > **(scroll down to **Stack trace**)**: This is an example of an stack trace that includes the option to resymbolicate the crash occurrence. To analyze and debug your source code using Xcode, select **Export crash details**.

Author: ```suggestion
        **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Crashes > Crash type >** (select a crash): This is an example of a stack trace that includes the option to upload a new Android map file or additional iOS .dSYM files. To analyze and debug your source code using Xcode, select **Export crash details**.
```
